examples/data-objects/inventory-app/webpack.config.cjs

```diff
@@ -38,26 +36,20 @@ module.exports = (env) => {
 			output: {
 				filename: "[name].bundle.js",
 				path: path.resolve(__dirname, "dist"),
-				library: "[name]",
-				// https://github.com/webpack/webpack/issues/5767
-				// https://github.com/webpack/webpack/issues/7939
-				devtoolNamespace: "fluid-experimental/tree-api",
-				libraryTarget: "umd",
+				library: { name: "[name]", type: "umd" },
 			},
-			plugins: [
-				new webpack.ProvidePlugin({
-					process: "process/browser",
-				}),
-			],
-			// This impacts which files are watched by the dev server (and likely by webpack if watch is true).
-			// This should be configurable under devServer.static.watch
-			// (see https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md) but that does not seem to work.
-			// The CLI options for disabling watching don't seem to work either, so this may be a symptom of using webpack4 with the newer webpack-cli and webpack-dev-server.
 			watchOptions: {
 				ignored: "**/node_modules/**",
 			},
 		},
-		isProduction ? require("./webpack.prod.cjs") : require("./webpack.dev.cjs"),
```

Comment:
I don't find having 3 webpack files useful, especially given that if you run `webpack --config webpack.prod.config` it doesn't work, since those files weren't actually webpack configs, but part of this one. I think its simpler an ess likley to confuse people to just inline them here.

Reply:
Fewer files === awesome. <3